### ISSUE TITLE
GREAT :heart_decoration: 

### ISSUE THREAD

**Issue by manhph2211** [tuanh123789/AdaSpeech#1]
Well, for me, this repo deserves thousands of stars :+1:

**Comment by LegendRGC** [comment]
> Well, for me, this repo deserves thousands of stars 👍

Is the model available，please？

**Comment by tuanh123789** [comment]
Yes i have pretrained model training on Vietnamese and English

**Comment by yiwei0730** [comment]
@tuanh123789 
Can I ask for the loss graph in the tensorboard? 
I have some problem in phoneme_level_prediction, the loss is always high, when i running the code which i wrote.

**Comment by tuanh123789** [comment]
@yiwei0730 Yes, I will update loss graph in tensorboard soon. In my experiments with both English and Vietnamese phoneme_level_prediction loss is high, but it still working. When phoneme_level_prediction loss convergence, it's about 1.5

**Comment by yiwei0730** [comment]
@tuanh123789  Thanks for your apply. Is your loss convergence says for the Validation data or the training data
For my experiments the training acoustic loss is in 3 and the validation acoustic loss is in 1.8, but maybe the difference with i did'nt use the masked part in the phoneme level loss

**Comment by yiwei0730** [comment]
There is an another problem with tour training, In AdaSpeech paper, the acoustic modeling part is list in phoneme level -> utterance level -> speaker embedding，but in your code you inheritance the list in "rishikksh github" using utterance level -> phoneme level -> speaker embedding，maybe this is a little bug in your code.

**Comment by LegendRGC** [comment]
> Yes i have pretrained model training on Vietnamese and English

thank you!

**Comment by LegendRGC** [comment]
> Yes i have pretrained model training on Vietnamese and English

Could you upload your current project include the "text grid folder" as a example,please？I can't run the "preprocess.py" because I don't know what I lack at present.thank you very much!

**Comment by LegendRGC** [comment]
> 
about english

**Comment by tuanh123789** [comment]
@LegendRGC ok, i will update data sample soon

**Comment by LegendRGC** [comment]
> @LegendRGC ok, i will update data sample soon
thank you very much,sir!

**Comment by LegendRGC** [comment]
> Yes i have pretrained model training on Vietnamese and English

Excuse me,how to get the "checkpoint" file,please?

**Comment by LegendRGC** [comment]
> > Yes i have pretrained model training on Vietnamese and English
> 
> Excuse me,how to get the "checkpoint" file,please?

the "vocoder checkpoint"

**Comment by tuanh123789** [comment]
> > Yes i have pretrained model training on Vietnamese and English
> 
> Could you upload your current project include the "text grid folder" as a example,please？I can't run the "preprocess.py" because I don't know what I lack at present.thank you very much!

I uploaded data sample in preprocessed_data and raw_data folder, please check and try again ^^

**Comment by tuanh123789** [comment]
> @tuanh123789 Can I ask for the loss graph in the tensorboard? I have some problem in phoneme_level_prediction, the loss is always high, when i running the code which i wrote.
I upload tensorboard for pretrain and finetune in Readme, please check ^^

**Comment by tuanh123789** [comment]
> @tuanh123789 Can I ask for the loss graph in the tensorboard? I have some problem in phoneme_level_prediction, the loss is always high, when i running the code which i wrote.

I upload tensorboard for pretrain and finetune in Readme, please check ^^

**Comment by yiwei0730** [comment]
Yes, I check your tensorboard, I've seen something interesting.
the phoneme level loss validation have the same result in my training. but the training loss is not the same. 
But I don't used the mask in this loss, since it make our acoustic loss more bigger. 
And your acoustic loss maybe is the correct, since my acoustic loss is increasing in training but your acoustic loss is decreasing.

**Comment by tuanh123789** [comment]
> Yes, I check your tensorboard, I've seen something interesting. the phoneme level loss validation have the same result in my training. but the training loss is not the same. But I don't used the mask in this loss, since it make our acoustic loss more bigger. And your acoustic loss maybe is the correct, since my acoustic loss is increasing in training but your acoustic loss is decreasing.

Yes, you can try masking to compute loss, hope it working ^^

**Comment by yiwei0730** [comment]
Thank you, I want to ask for the phoneme level loss, did you have some idea for decline the loss value?

**Comment by arampacha** [comment]
> Yes i have pretrained model training on Vietnamese and English

Hi, thanks for your work!
Are there pretrained models available already? Could you point me to checkpoints pls?

**Comment by zaverichintan** [comment]
I started training (pretraining) using LibriTTS dataset. 
I see the following behaviour:
<img width="1443" alt="Screenshot 2022-08-16 at 03 47 13" src="https://user-images.githubusercontent.com/11918544/184861583-0f8250a1-2611-4c30-bfdc-4213a667da61.png">
 
 Should I change the batch size or any other parameters?

**Comment by vedantk-b** [comment]
Hey, if anyone has some pretrained weights, can they share a drive link or something to that?
